### ISSUE TITLE
[Bug]: Playwright times out when trying to take a screenshot while headed (not headless)

### ISSUE THREAD

**Issue by dosolkowski-work** [microsoft/playwright-dotnet#2950]
### Version

1.44.0

### Steps to reproduce

Using Microsoft.Playwright.NUnit, in various places in our tests we attempt to capture a screenshot using `await page.ScreenshotAsync(new PageScreenshotOptions { Path = path });` (where `page` is a Playwright page, and `path` is the name of the PNG file we want written).

### Expected behavior

We expect a screenshot to be written, or at least some sort of error message that explains why it could not be.

### Actual behavior

The first time we try to capture a screenshot, it times out with a bit of log information:

```
    2024-06-28T14:13:37.0594483-04:00 [Error] TestContext.Project.ReleaseTests: [0]: failed to capture screen shot System.TimeoutException: Timeout 14500ms exceeded.
Call log:
  - taking page screenshot
  - waiting for fonts to load...
   at Microsoft.Playwright.Transport.Connection.InnerSendMessageToServerAsync[T](ChannelOwner object, String method, Dictionary`2 dictionary, Boolean keepNulls) in /_/src/Playwright/Transport/Connection.cs:line 228
   at Microsoft.Playwright.Transport.Connection.WrapApiCallAsync[T](Func`1 action, Boolean isInternal) in /_/src/Playwright/Transport/Connection.cs:line 532
   at Microsoft.Playwright.Core.Page.ScreenshotAsync(PageScreenshotOptions options) in /_/src/Playwright/Core/Page.cs:line 675
   at Project.ReleaseTests.E2E.PageExtensions.ScreenshotAsync(IPage page, String path, ILogger logger) in C:\Work\Source\test\Project.ReleaseTests\E2E\PageExtensions.cs:line 36
```

Subsequent attempts just time out:

```
2024-06-28T14:13:51.5863866-04:00 [Error] TestContext.Project.ReleaseTests: [0]: failed to capture screen shot System.TimeoutException: Timeout 14500ms exceeded.
   at Microsoft.Playwright.Transport.Connection.InnerSendMessageToServerAsync[T](ChannelOwner object, String method, Dictionary`2 dictionary, Boolean keepNulls) in /_/src/Playwright/Transport/Connection.cs:line 228
   at Microsoft.Playwright.Transport.Connection.WrapApiCallAsync[T](Func`1 action, Boolean isInternal) in /_/src/Playwright/Transport/Connection.cs:line 532
   at Microsoft.Playwright.Core.Page.ScreenshotAsync(PageScreenshotOptions options) in /_/src/Playwright/Core/Page.cs:line 675
   at Project.ReleaseTests.E2E.PageExtensions.ScreenshotAsync(IPage page, String path, ILogger logger) in C:\Work\Source\test\Project.ReleaseTests\E2E\PageExtensions.cs:line 36
```

### Additional context

These failures only happen while headless is `false`. If I set headless to `true` the screenshots are taken fine.

### Environment

```Text
- Operating System: Windows 10
- CPU: Intel Core i9
- Browser: Chromium
- .NET Version (TFM): SDK 6.0.403, runtime 6.0.31
- Other info:
```

**Comment by mxschmitt** [comment]
This should be fixed as part of v1.45.0 which we'll release probably today or tomorrow.

**Comment by mxschmitt** [comment]
Feel free to re-file if its not fixed in v1.45.0

**Comment by dosolkowski-work** [comment]
Confirming the problem is resolved in 1.45. Thanks!
